user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/ProteomicsMeasurementEntry.java

```diff
@@ -0,0 +1,30 @@
+package life.qbic.datamanager.views.projects.project.measurements;
+
+/**
+ * <b><class short description - 1 Line!></b>
```

Comment:
please provide a description, especially about what this record is needed for that wasn't there for ptxMeasurement registration